src/edu/csus/ecs/pc2/core/imports/clics/CLICSAward.java

```diff
@@ -52,11 +50,11 @@ public CLICSAward() {
      * @param citation ex. Contest winner
      * @param team_ids lsit of ids
      */
-    public CLICSAward(String id, String citation, List<String> team_ids) {
+    public CLICSAward(String id, String citation, String [] team_ids) {
```

Comment:
This change, which apparently was made in response to my previous review comment, is in effect the exact _opposite_ of what I suggested/requested.  Previously, the constructor accepted a `List<String>` for `team_ids`.  As I said in my previous comment, this seems correct; external clients should know only that they are to provide a `List` and should not know how that `List` is to be stored externally. The problem I pointed out previously was in the method `getTeamIds()`, which was (and still is) returning type `String[]`.  This means there are now TWO places where the internal storage type (specifically, an _array_) is exposed to external clients.

The request was to change the return type of method `getTeamIds()` to be `List<String>`.  This commit does the opposite of that.

Reply:
As of the latest commit, the return type of getTeamIds() and the argument to constructor are the same String [].  I think this is fine and consistent.  IMHO I don't think should block approval.

Reply:
I again fall back to my teaching days, where among other things I tried to pound into my student's heads the notion that a class should not expose to outside clients the types which it uses for internal storage.  For example, a class `Stack` should not allow external users of a `Stack` to know whether the Stack is implemented as an array, a linked list, a hashtable, or using some other data type.  The reason for this is because if clients know the internal data type, they can make code assumptions based on that type; if the original class later decides to change the storage type (for efficiency reasons, for example), that will _break external clients_.  Making internal changes to a class shouldn't do that.

In this case, the use of `String []` parameters and return types is allowing external clients to know that these values are (always) stored as an _array of Strings_.  If we later decide that it would be better to store things internally as (say) a `Vector<String>`, making such a change will break external clients.

This is the whole purpose of the `List<E>` interface in Java -- it allows specification of an ordered list of things without exposing the internal implementation of the list; this in turn makes for more robust code.

I stand by my original request that the constructors and the accessor methods use `List<String>` instead of `String[]` as their parameters and return types.   

Yes, I'm being pedantic -- but with good reason in this case IMHO.

Reply:
Yeah, I think this is one of those "religious" things... I think we can agree to disagree here.  By using `String[]`, this (IMO) does not conveying any information about how the information is stored in the object.  If, for example, `private String [] team_ids`; was `public String [] team_ids`; then yes, this does expose the internals.  Just because an array of strings is passed in the constructor and returned as an array in what is presumably an accessor, does not necessarily mean it is stored or used that way in the `CLICSAwards` object, right?  Also, if, for some reason, the data that is passed in or returned changes from something OTHER than a "bunch of" of `String` objects, client code will have to be modified _anyway_ to accommodate the new type of object.  `List<String>`  or `String[] `or `Vector<String>`, any of those work for _me_.

Reply:
Ok, I'll say two things.

One:  I still think I'm right and it's more than a "religious thing".  There is a good reason the designers of Java added the `Collection` interface to the language, and added `List` which inherits from (extends) collection, and added `ArrayList` (which implements `List`):  it was to support the ability to create abstractions in the types which are passed between classes.  I think that applies here.

Two:  people can have different opinions on stuff like this, and it's not worth spending more time on in this case.   Accordingly, I withdraw my request that the code be changed, and I will mark this conversation (and the related ones) "resolved" so we can move on.